content/scalers/redis-streams.md

```diff
@@ -0,0 +1,147 @@
+    pendingEntriesCount: "10" # Required - number of entries in the Pending Entries List for the specified consumer group in the Redis Stream
+    enableTLS: "false" # optional
+    databaseIndex: "0" # optional
+```
+
+**Parameter list:**
+
+- `address`: Name of the environment variable your deployment uses to get the Redis server URL. The resolved host should follow a format like `my-redis:6379`.
+   - This is usually resolved from a `Secret V1` or a `ConfigMap V1` collections. `env` and `envFrom` are both supported.
+
+> As an alternative to the `address` field, the user can specify `host` and `port` parameters. 
+
+- `host`: Name of the environment variable your deployment uses to get the Redis server host. 
+    - This is usually resolved from a `Secret V1` or a `ConfigMap V1` collections. `env` and `envFrom` are both supported.
+
+> It is not required if `address` has been provided
+
+- `port`: Name of the environment variable your deployment uses to get the Redis server port. 
+   - This is usually resolved from a `Secret V1` or a `ConfigMap V1` collections. `env` and `envFrom` are both supported.
+
+> It is only to be used along with the `host` attribute and not required if `address` has been provided
+
+- `password` (optional): Name of the environment variable your deployment uses to get the Redis password.
+   - This is usually resolved from a `Secret V1` or a `ConfigMap V1` collections. `env` and `envFrom` are both supported.
+
+- `stream`: Name of the Redis Stream
+
+- `consumerGroup`: Name of the Consumer group associated with Redis Stream
+
+- `pendingEntriesCount`: Threshold for the number of `Pending Entries List`. This is the average target value to scale the workload. Defaults to `5` 
+
+- `databaseIndex`: The Redis database index. Defaults to `0` if not specified
+
+- `enableTLS`: Set this to `true` if TLS connection to Redis is required. Defaults to `false`
+
+### Authentication Parameters
+
+The scaler supports two modes of authentication:
+
+**`password` metadata field**
+
+Use the `password` field in the `metadata` to specify the name of an environment variable that your deployment uses to get the Redis password.
+
+This is usually resolved from a `Secret V1` or a `ConfigMap V1` collections. `env` and `envFrom` are both supported.
+
+Here is an example:
+
+```yaml
+apiVersion: keda.k8s.io/v1alpha1
+kind: ScaledObject
+metadata:
+  name: redis-streams-scaledobject
+  namespace: default
+  labels:
+    deploymentName: redis-streams-consumer
+spec:
+  scaleTargetRef:
+    deploymentName: redis-streams-consumer
+  pollingInterval: 15
+  cooldownPeriod: 200
+  maxReplicaCount: 25
+  minReplicaCount: 1
+  triggers:
+    - type: redis-streams
+      metadata:
+        address: REDIS_HOST
+        password: REDIS_PASSWORD # name of the environment variable in the Deployment
+        stream: my-stream
+        consumerGroup: consumer-group-1
+        pendingEntriesCount: "10"
+```
+
+**Using `TriggerAuthentication`**
```

Comment:
Suggestion - Use header so that it shows up in side navigation
```suggestion
#### Using `TriggerAuthentication`
```

Reply:
updated as per feedback